components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/operator-finder/operator-finder.js

```diff
@@ -309,11 +309,11 @@ define(['jquery', 'lodash', 'log','remarkable', 'handlebar', 'designViewUtils','
         /**
          * functions to get the not installed extension array details.
          */
-        var getNotInstalledExtensionDetails = function () {
+        var getUninstalledExtensionDetails = function () {
```

Comment:
@dnwick & @SinthujanSintha Shall we say `getNonInstalledExtensionDetails` ? 'Uninstalled' might give the wrong idea as "Installed, and removed later"

Reply:
+1